src/me/eglp/gv2/commands/fun/CommandReminder.java

```diff
@@ -0,0 +1,191 @@
+package me.eglp.gv2.commands.fun;
+
+import java.awt.Color;
+import java.time.Instant;
+import java.time.LocalDateTime;
+import java.time.ZoneOffset;
+import java.time.format.DateTimeFormatter;
+import java.time.format.DateTimeParseException;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.List;
+import java.util.stream.Collectors;
+
+import me.eglp.gv2.util.GraphiteUtil;
+import me.eglp.gv2.util.base.GraphiteMessageChannel;
+import me.eglp.gv2.util.base.guild.GraphiteGuildMessageChannel;
+import me.eglp.gv2.util.base.guild.GraphiteModule;
+import me.eglp.gv2.util.base.guild.reminder.GuildReminder;
+import me.eglp.gv2.util.base.guild.reminder.ReminderRepetitionEnum;
+import me.eglp.gv2.util.command.Command;
+import me.eglp.gv2.util.command.CommandCategory;
+import me.eglp.gv2.util.command.CommandInvokedEvent;
+import me.eglp.gv2.util.command.ParentCommand;
+import me.eglp.gv2.util.emote.JDAEmote;
+import me.eglp.gv2.util.emote.unicode.GraphiteUnicode;
+import me.eglp.gv2.util.lang.DefaultLocaleString;
+import me.eglp.gv2.util.lang.DefaultMessage;
+import me.eglp.gv2.util.lang.GraphiteTimeParser;
+import me.eglp.gv2.util.lang.defaults.DefaultLocale;
+import me.eglp.gv2.util.lang.defaults.DefaultLocales;
+import net.dv8tion.jda.api.EmbedBuilder;
+import net.dv8tion.jda.api.entities.Message;
+import net.dv8tion.jda.api.entities.channel.ChannelType;
+import net.dv8tion.jda.api.entities.emoji.Emoji;
+import net.dv8tion.jda.api.interactions.commands.OptionType;
+import net.dv8tion.jda.api.interactions.commands.build.OptionData;
+
+/**
+ * This is the reminder Command class. It handles the different sub-commands and deligates the functionality to {@link GuildReminder}.
+ * 
+ * @author The Arrayser
+ * @date 2023.03
```

Comment:
I don't think that's a full date

Reply:
That's because, these files have been developed on multiple days in the month of march...

Reply:
I think it should be the creation date of the file